### ISSUE TITLE
mapzen.places.search different result count compared to spelunker

### ISSUE THREAD

**Issue by simonw** [whosonfirst/whosonfirst-www-api#116]
This page of the spelunker claims to list 1,429 results: https://whosonfirst.mapzen.com/spelunker/placetypes/neighbourhood/?locality_id=-1&iso=us

The same search against the API (I think this is the same search) says this instead: https://places.mapzen.com/v1?method=mapzen.places.search&iso=us&locality_id=-1&placetype=neighbourhood&api_key=mapzen-xxx

    "total": 44889,
    "pages": 449,
    "per_page": 100

Is this an API bug or am I doing something wrong?

**Comment by thisisaaronland** [comment]
Yeah, that's weird. I will poke around. I'm guessing that it's probably a disconnect in the way the query is being assembled in the API...
